src/internal/DotStarEsp32DmaSpiMethod.h

```diff
@@ -116,7 +116,8 @@ template<typename T_SPISPEED, typename T_SPIBUS> class _DotStarEsp32DmaSpiMethod
         //Initialize the SPI bus
         ret = spi_bus_initialize(T_SPIBUS::SpiHostDevice, &buscfg, SPI_DMA_CH_AUTO);
         ESP_ERROR_CHECK(ret);
-
+        
+        memset(&_spiTransaction, 0, sizeof(spi_transaction_t));
```

Comment:
in general, for allocated memory initialization memset is used, but for all other they should just be initialized normally in the constructor unless there is a specific reason. 
```
_spiTransaction = {0}; 
```
or as an initializer on constructor
```
_spiTransaction(0),
```

Reply:
I totally missed this! thanks for the feedback, glad you cleaned it up and merged 🙏